patches/tModLoader/Terraria/ModLoader/ModMenu.cs

```diff
@@ -94,4 +96,36 @@ public virtual void Update(bool isOnTitleScreen)
 	public virtual void PostDrawLogo(SpriteBatch spriteBatch, Vector2 logoDrawCenter, float logoRotation, float logoScale, Color drawColor)
 	{
 	}
+
+	/// <summary>
+	/// Called just before the sky is drawn, and allows you to modify some of the parameters of the sky draw code.
+	/// <br>Return false to prevent the usual sky from drawing.</br>
+	/// </summary>
+	public virtual bool PreDrawSky(SpriteBatch spriteBatch, ref bool drawStars, ref bool drawSunAndMoon)
+	{
```

Comment:
Create a `SkyDrawParams` struct for `drawStars` and `drawSunAndMoon`, similar to `BossBarDrawParams`, so that future upgrades to this hook do not change its signature.